File: add/codesnippet/CSharp/m-system.net.networkinfo_6_1.cs
         public static void ShowActiveTcpListeners()
         {
             Console.WriteLine("Active TCP Listeners");
             IPGlobalProperties properties = IPGlobalProperties.GetIPGlobalProperties();
             IPEndPoint[] endPoints =  properties.GetActiveTcpListeners();
             foreach (IPEndPoint e in endPoints)
             {
                 Console.WriteLine(e.ToString());
             }
         }

File: add/codesnippet/CPP/m-system.net.networkinfo_6_1.cpp
void ShowActiveTcpListeners()
{
   Console::WriteLine( "Active TCP Listeners" );
   IPGlobalProperties ^ properties = IPGlobalProperties::GetIPGlobalProperties();
   array<IPEndPoint^>^endPoints = properties->GetActiveTcpListeners();
   System::Collections::IEnumerator^ myEnum7 = endPoints->GetEnumerator();
   while ( myEnum7->MoveNext() )
   {
      IPEndPoint^ e = safe_cast<IPEndPoint^>(myEnum7->Current);
      Console::WriteLine( e );
   }
}